packages/obojobo-document-engine/ObojoboDraft/Chunks/Question/editor.js

```diff
@@ -23,14 +23,34 @@ const plugins = {
 	schema: Schema
 }
 
-Common.Store.registerEditorModel('ObojoboDraft.Chunks.Question', {
+Common.Registry.registerModel('ObojoboDraft.Chunks.Question', {
 	name: 'Question',
 	icon: Icon,
 	isInsertable: true,
 	insertJSON: emptyNode,
 	slateToObo: Converter.slateToObo,
 	oboToSlate: Converter.oboToSlate,
-	plugins
+	plugins,
+	getNavItem(model) {
```

Comment:
This is the same question as the heading: since we are not displaying this in the navigation, do we really want to register the nav item?